lib/stash_web/controllers/stash_html/index.html.heex

```diff
@@ -0,0 +1,22 @@
+<h1> Hello! </h1>
+
+        v<%= Application.spec(:phoenix, :vsn) %>
+
+<div id="index">
+    <table>
+      <thead>
+        <tr>
+          <th>Key</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        <%= for {key, value} <- @stash do %>
```

Comment:
good, but see if https://hexdocs.pm/phoenix/components.html#html-extensions is better for this?

```
<ul>
  <li :for={item <- @items}><%= item.name %></li>
</ul>
```
this list comprehension is more efficient at runtime, which, admittedly, for our general use case is not particularly important :-)